tephra-core/src/main/java/org/apache/tephra/distributed/TransactionServiceClient.java

```diff
@@ -130,10 +134,11 @@ public static void doMain(boolean verbose, Configuration conf) throws Exception
    * for service discovery. Otherwise it will look for the port in the
    * config and use localhost.
    * @param config a configuration containing the zookeeper properties
+   * @param clientId id of the client that identifies it when it starts a transaction
    */
   @Inject
-  public TransactionServiceClient(Configuration config,
-                                  ThriftClientProvider clientProvider) {
+  public TransactionServiceClient(Configuration config, ThriftClientProvider clientProvider,
```

Comment:
Some clients like Apache Phoenix do not use the guice modules. They instantiate `TransactionServiceClient` directly. It will be good to have the existing constructor too and assign the default value for the clientId in it.